make-release.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# Release process automation script.
+# Used to create branch/tag, update VERSION files
+# and and trigger release by force pushing changes to the release branch
```

Comment:
this comment is not correct. 

How about
```
Used to create a branch update the version stored in package.json, and push a tag
Once the tag is created, image-publish.yml will be triggered to build the container image.
```

Reply:
thx